DNN Platform/Library/Common/Utilities/DataCache.cs

```diff
@@ -211,6 +211,7 @@ public class DataCache
         public const string UserProfileCacheKey = "UserProfile|{0}|{1}";
         public const int UserProfileCacheTimeOut = UserCacheTimeOut;
 
+        public const string UserNotificationsConversationCountCacheKey = "UserNitifConversationCount|{0}|{1}";
```

Comment:
This is intentional and any page transitions withing 30 seconds will save some additional SQL requests.
This doesn't need any synchronization as it is meant to live for short time without the need to synchronize with other servers' caches.